src/CppUTestExt/MockFailure.cpp

```diff
@@ -104,10 +104,15 @@ MockExpectedCallsDidntHappenFailure::MockExpectedCallsDidntHappenFailure(UtestSh
 MockUnexpectedCallHappenedFailure::MockUnexpectedCallHappenedFailure(UtestShell* test, const SimpleString& name, const MockExpectedCallsList& expectations) : MockFailure(test)
 {
     int amountOfExpectations = expectations.amountOfExpectationsFor(name);
-    if (amountOfExpectations)
```

Comment:
I'm not opposed to removing them, but I think it slightly improves readability and doesn't hurt generated code.

If you're suggesting calling amountOfExpectationsFor() twice, I suppose it won't be an issue, since it's a const method.  The compiler will elide the second call, correct?